### ISSUE TITLE
Test failures in ejb30/lite/stateful/timeout

### ISSUE THREAD

**Issue by alwin-joseph** [eclipse-ee4j/jakartaee-tck#1028]
**Describe the bug**

44 tests are failing in ejb30/lite/stateful/timeout (vehicles ejbembed & ejblitejsp) in Platform TCK against Glassfish 7(latest nightly)

Latest job run url:  https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/master/1834.

Tests failing : 

com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.dayUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.dayUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.defaultUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.defaultUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.defaultUnitNoInterface_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.defaultUnitNoInterface_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.hourUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.hourUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.microsecondUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.microsecondUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.millisecondUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.millisecondUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.minus1Timeout_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.minus1Timeout_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.nanosecondUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.nanosecondUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.secondUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.secondUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.secondUnitNoInterface_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.secondUnitNoInterface_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.zeroTimeout_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.annotated.Client.zeroTimeout_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.dayUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.dayUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.defaultUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.defaultUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.defaultUnitNoInterface_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.defaultUnitNoInterface_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.hourUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.hourUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.microsecondUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.microsecondUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.millisecondUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.millisecondUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.minus1Timeout_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.minus1Timeout_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.nanosecondUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.nanosecondUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.secondUnitLocal_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.secondUnitLocal_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.secondUnitNoInterface_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.secondUnitNoInterface_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.zeroTimeout_from_ejbembed
com.sun.ts.tests.ejb30.lite.stateful.timeout.descriptor.Client.zeroTimeout_from_ejblitejsp

@starksm64 @scottmarlow @gurunrao

**Comment by dmatej** [comment]
Should be resolved now as I passed with current master of GlassFish 7 whole ejb30 (built from yesterday's TCK master) this morning. I can't wait it to be confirmed by Jenkins :-)

```
[runcts] OUT => [javatest.batch] ********************************************************************************
[runcts] OUT => [javatest.batch] Completed running 3956 tests.
[runcts] OUT => [javatest.batch] Number of Tests Passed      = 3956
[runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
[runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
[runcts] OUT => [javatest.batch] ********************************************************************************

[runcts] OUT => [javatest.batch] Report written to /repo/git/jakartaee-tck-runner/jakartaeetck-report/ejb30
[runcts] OUT =>
[runcts] OUT => BUILD SUCCESSFUL
[runcts] OUT => Total time: 924 minutes 24 seconds
[runcts] Proc status ==> "0"
[runcts] Completed "/repo/git/jakartaee-tck-runner/jakartaeetck/src/com/sun/ts/tests/ejb30"

BUILD SUCCESSFUL
Total time: 924 minutes 37 seconds
```

**Comment by scottmarlow** [comment]
> Should be resolved now as I passed with current master of GlassFish 7 whole ejb30 (built from yesterday's TCK master) this morning. I can't wait it to be confirmed by Jenkins :-)
> 
> ```
> [runcts] OUT => [javatest.batch] ********************************************************************************
> [runcts] OUT => [javatest.batch] Completed running 3956 tests.
> [runcts] OUT => [javatest.batch] Number of Tests Passed      = 3956
> [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
> [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
> [runcts] OUT => [javatest.batch] ********************************************************************************
> 
> [runcts] OUT => [javatest.batch] Report written to /repo/git/jakartaee-tck-runner/jakartaeetck-report/ejb30
> [runcts] OUT =>
> [runcts] OUT => BUILD SUCCESSFUL
> [runcts] OUT => Total time: 924 minutes 24 seconds
> [runcts] Proc status ==> "0"
> [runcts] Completed "/repo/git/jakartaee-tck-runner/jakartaeetck/src/com/sun/ts/tests/ejb30"
> 
> BUILD SUCCESSFUL
> Total time: 924 minutes 37 seconds
> ```

Were you running Java SE 11 or SE 17?  We seem to only see the test failure on Java SE 11.

**Comment by dmatej** [comment]
Sorry for my late response, I was ill, now I am getting better.

I used JDK11, today I analysed some logs from Jenkins, it tested different GF versions. We discussed that with @arjantijms , tomorrow we will know more (how to pass both CDI and EJB tests).

- GF built for commit a909e10 passed ejb30 on JDK11 on my computer
- GF built for commit 22e99bf056 passed on JDK17 on Eclipse's Jenkins
- GF built for commit 1d3180 failed on JDK11 on Eclipse's Jenkins
